app/src/main/java/sdp/moneyrun/Game.java

```diff
@@ -222,26 +296,14 @@ public void setPlayers(List<Player> p){
             gameDbData.setPlayers(p);
         }else{
             gameDbData.setPlayers(p);
-            rootReference.child("open_games").child(id).child("players").setValue(p);
+            FirebaseDatabase.getInstance().getReference()
```

Comment:
I think we will be using this quite often from now on, maybe for next week create a function for it? What do you think?

Reply:
Indeed, we just need to be careful for testing because getInstance cannot be called in regular unit tests.

Reply:
create a function for what now?